BigQuery content edits (#2248)
segmentio/segment-docs: src/connections/storage/catalog/bigquery/index.md

Using BigQuery through Segment means you'll get a fully managed data pipeline
loaded into one of the most powerful and cost-effective data warehouses today.

## Getting Started

> In order store your Segment data in BigQuery, you must [enable BigQuery for your Google Cloud project](#create-a-project-and-enable-bigquery), [create a GCP service account for Segment to assume](#create-a-service-account-for-segment), and [create a warehouse in the Segment app](#create-the-warehouse-in-segment).

Reviewer: ```suggestion
To store your Segment data in BigQuery, complete the following steps:
- [Enable BigQuery for your Google Cloud project](#create-a-project-and-enable-bigquery)
- [Create a GCP service account for Segment to assume](#create-a-service-account-for-segment)
- [Create a warehouse in the Segment app](#create-the-warehouse-in-segment).
```

Personal preference here, as I usually find steps more easily readable in lists vs sentences. Either way, I think we can remove "In order to" and "you must". Those don't do much work in this sentence.

### Create a Project and Enable BigQuery

1. Navigate to the [Google Developers Console](https://console.developers.google.com/)
2. Configure [Cloud Platform](https://console.cloud.google.com/):
  - If you don't have a project already, [create one](https://support.google.com/cloud/answer/6251787?hl=en&ref_topic=6158848).
  - If you have an existing project, you will need to [enable the BigQuery API](https://cloud.google.com/bigquery/quickstart-web-ui).
    Once you've done so, you should see BigQuery in the "Resources" section of Cloud Platform.
  - **Note:** make sure [billing is enabled](https://support.google.com/cloud/answer/6293499#enable-billing) on your project,

>     or Segment will not be able to write into the cluster.

Reviewer: This should probably be part of line 30.

> 3. Copy your project ID, as you will need it when creating your warehouse source in the Segment app.

Reviewer: ```suggestion
3. Copy the project ID. You will need it when you create a warehouse source in the Segment app.
```

1. In Segment, go to **Workspace** > **Add destination** > Search for "BigQuery"
2. Select **BigQuery**
3. Add a name for the destination to the **Name your destination** field
4. Enter your project ID in the **Project** field
5. Copy the contents of the credentials (the JSON key) into the **Credentials** field <br/>
**Optional:** Enter a [region code](https://cloud.google.com/compute/docs/regions-zones/) in the **Location** field (the default will be "US")
6. Click **Connect**

> 7. if Segment is able to successfully connect with the provided **Project ID** and **Credentials**,

Reviewer: ```suggestion
7. If Segment can connect with the provided **Project ID** and **Credentials**,
```

Can make this more active-voice. "Successfully" is also a bit of a weasel word.

> a warehouse will be created and your first sync should begin shortly

Reviewer: This should also probably be on line 57.

For early customers using BigQuery with Segment, rather than providing Segment
with credentials, access was granted to a shared Service Account
(`connector@segment-1119.iam.gserviceaccount.com`). While convenient for early
adopters, this presented potential security risks that Segment would prefer to address
proactively.

As of **March 2019**, Segment requires BigQuery customers to
create their own Service Accounts and provide the app with those credentials instead.
In addition, any attempts to update warehouse connection settings will also
require these credentials. This effectively deprecates the shared Service
Account.

> In order to stay ahead of this change, make sure to migrate your warehouse by following

Reviewer: ```suggestion
To stay ahead of this change,  migrate your warehouse by following
```
